XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -214,4 +214,8 @@
   <ItemGroup>
     <Folder Include="Views\Snackbar\Helpers\WPF\" />
   </ItemGroup>
+
```

Comment:
probably it should be removed

Reply:
Vlad is right. Discard this file please

Reply:
Every time I add a class to this project, it gets added like this and I have to manually set the Build Action to c# compiler. Do you guys have any idea why that is? As this time I forgot to double check the project file. Anyway, I've removed this 👍